src/main/flight/pid.c

```diff
@@ -249,6 +249,11 @@ void INIT_CODE pidInitProfile(const pidProfile_t *pidProfile)
     acroTrainerInit(pidProfile);
 #endif
     rescueInitProfile(pidProfile);
+
```

Comment:
Move this before "Initialise sub-profiles"

Reply:
Done